### ISSUE TITLE
Serialization of attr.ib which is is a list of attr.s fails

### ISSUE THREAD

**Issue by hansonap** [drorvinkler/yasoo#1]
```
from typing import List

import attr
from yasoo import serialize


@attr.s
class Bar:
    name = attr.ib()
    type = attr.ib()
    value = attr.ib()


@attr.s
class Foo:
    bars = attr.ib(type=List[Bar])


serialize(Foo([Bar('a', 'b', 'c'), Bar('a', 'b', 'c')]))

```

TypeError: Found object of type "Bar" which cannot be serialized

**Comment by drorvinkler** [comment]
Thank you for pointing that out.

This issue is now fixed (see [9253e2b85d1590791db3d086f0dfa82d0519ecd7](https://github.com/drorvinkler/yasoo/commit/9253e2b85d1590791db3d086f0dfa82d0519ecd7)).
Please update to version 0.2.1 and let me know if it works as needed.

**Comment by hansonap** [comment]
Verified, the issue is fixed - thanks!
